common/src/main/java/net/william278/huskhomes/command/BackCommand.java

```diff
@@ -36,6 +36,7 @@ public class BackCommand extends InGameCommand {
     protected BackCommand(@NotNull HuskHomes plugin) {
         super("back", List.of(), "", plugin);
         addAdditionalPermissions(Map.of("death", false));
+        addAdditionalPermissions(Map.of("previous", false));
```

Comment:
```suggestion
        addAdditionalPermissions(Map.of(
            "death", false,
            "previous", false
        ));
```
`addAdditionalPermissions` accepts a map, so we can simply add more pairs of values to the first `Map#of` call to generate a map with two key-value pairs :)

Reply:
Nice catch! Done

But I have another concern about the permission resolution (I've commeted it on this issue)